app/src/main/java/edu/uc/hornunjb/weatherwear/repo/repository.kt

```diff
@@ -6,7 +6,11 @@ import edu.uc.hornunjb.weatherwear.utils.Constants
 import retrofit2.Call
 
 class Repository {
-    suspend fun getPost() : WeatherResponse {
+    /*
+     This returns the weather of the location found in the constants class
+     @return a weather response object for the call
+     */
+    suspend fun getWeather() : WeatherResponse {
```

Comment:
I renamed this function for the reasons stated in my earlier comments. I also added some KDocs as this function is fairly important. I used the kdoc syntax described here: https://kotlinlang.org/docs/kotlin-doc.html.